exercises/all-your-base/package.json

```diff
@@ -9,22 +9,70 @@
     "url": "https://github.com/exercism/xecmascript"
   },
   "devDependencies": {
-    "babel-cli": "^6.24.1",
-    "babel-core": "^6.24.1",
-    "babel-plugin-transform-object-rest-spread": "^6.23.0",
-    "babel-polyfill": "^6.23.0",
+    "babel-jest": "^19.0.0",
     "babel-preset-env": "^1.3.3",
-    "babel-preset-es2015": "^6.24.1",
-    "del": "~2.0.2",
-    "gulp": "~3.9.0",
-    "gulp-babel": "^6.1.2",
-    "gulp-eslint": "^1.1.0",
-    "gulp-jasmine": "~2.4.2",
-    "yargs": "~3.27.0"
+    "eslint": "^3.19.0",
+    "jest": "^19.0.2"
+  },
+  "jest": {
+    "modulePathIgnorePatterns": [
```

Comment:
This was needed to prevent Jest from warning about multiple `package.json` files in the project tree.